src/strategies/radicle-funding-nft/index.ts

```diff
@@ -0,0 +1,53 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const FUNDING_SUBGRAPH_URL = {
+  '1': 'https://api.studio.thegraph.com/query/9578/funding-subgraph-v4/v0.0.2'
```

Comment:
Is this graph a standard for radicle funding NFT? maybe we should have a way to pass this from params (so users can their own subgraph)

Reply:
@ChaituVR It's fine, this strategy and subgraph are specific to Radicle, the url can be hardcoded.

Reply:
I asked them if it's a standard subgraph, no response yet